easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED-2.1.3.eb

```diff
@@ -0,0 +1,41 @@
+name = 'CP2K'
+version = '20150904'
+versionsuffix = '-PLUMED-2.1.3'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+# we use the git mirror: this is r15816 or 68331ef35b7
+sources = ['68331ef35b7.zip']
+source_urls = ['https://github.com/cp2k/cp2k/archive/']
+
+patches = [
+    'CP2K-%(version)s-ifort-compiler-bug-fix.patch',
+    'CP2K-2.4.0-fix_compile_date_lastsvn.patch',
+]
+
+unpack_options = ' && mv cp2k-68331* cp2k'
```

Comment:
add a clarifying comment please, mention the error you're dancing around here (`Argument list too long`?)